packaging/python/build_wheels.bash

```diff
@@ -132,7 +133,7 @@ build_wheel_osx() {
     (( $skip )) && return 0
 
     echo " - Installing build requirements"
-    pip install -U delocate -r packaging/python/build_requirements.txt
+    pip install -U 'delocate<0.10.4' -r packaging/python/build_requirements.txt
```

Comment:
Any particular reason why we're explicit about the version here?

Reply:
Good catch! It's a leftover from an earlier attempt to get the wheels to build on my mac that is now obsolete.